components/src/dynamo/sglang/tests/test_sglang_gms_memory_saver.py

```
@@ -0,0 +1,149 @@
+        self.calls.append(("reallocate_all_handles", tag))
+
+    def remap_all_vas(self) -> None:
+        self.calls.append("remap_all_vas")
+        self.is_unmapped = False
+
+@pytest.fixture
+def build_impl(monkeypatch):
+    monkeypatch.setattr(
+        gms_memory_saver,
+        "get_socket_path",
+        lambda device_index, tag: f"/tmp/gms-test-{device_index}-{tag}.sock",
```

Ruff (0.15.9)

[error] Probable insecure usage of temporary file or directory: "/tmp/gms-test-"

(S108)